package.json

```diff
@@ -39,8 +39,10 @@
     "raw-loader": "^0.5.1",
     "rollup": "^0.52.1",
     "rollup-plugin-babel": "^3.0.2",
+    "rollup-plugin-commonjs": "^8.2.6",
     "rollup-plugin-html": "^0.2.1",
     "rollup-plugin-json": "^2.3.0",
+    "rollup-plugin-node-resolve": "^3.0.0",
```

Comment:
I had to add both of these plugins to get `lodash/debounce` into the bundles created by rollup... I am pretty new to rollbar so let me know if there is a better way